CHANGELOG.md

```diff
@@ -4,38 +4,51 @@
 
 ### New features
 
-#### Darkened secondary text colour
+#### The secondary text colour is now darker
```

Comment:
Do we need to link to the pull request where this was changed? (we seem to do this for the other entries)

Reply:
Thanks Vanita - added in 9892ac3